projects/components/src/public-api.ts

```diff
@@ -102,6 +102,18 @@ export * from './feature-check/feature-config-check.module';
 export * from './feature-check/feature.pipe';
 export * from './feature-check/if-feature.directive';
 
+// File Display
+export * from './file-display/file-item';
+export * from './file-display/file-display.component';
+export * from './file-display/file-display.module';
+
+// File Upload
+export * from './file-upload/drop-zone/drop-zone.directive';
```

Comment:
We don't need to export `file-display.component`, `file-display.module` and `drop-zone.directive` because they are internal details.

Reply:
Done